tools/RAiDER/delay.py

```diff
@@ -96,10 +100,11 @@ def tropo_delay_cube(dt, weather_model_file, ll_bounds, heights, los, out_proj=4
     # If output is desired in degrees
     use_weather_model_cube = False
 
-    if (cube_spacing_m is not None) or (crs != CRS(4326)):
+    if (cube_spacing_m is None) and (crs == CRS(4326)):
 
         use_weather_model_cube = True
-
+    
+    else:
```

Comment:
Should we not pass the default cube spacing to be the approximate model spacing? Ideally even its actual grid instead of doing interpolations?